tests/h/search/query_test.py

```diff
@@ -391,17 +391,22 @@ def search(self, search):
         return search
 
 
-class TestUriFilter(object):
```

Comment:
Here we are just moving all the old tests from the UriFilter into the UriCombinedWildcardFilter.